src/scripts/restore-latest.sh

```diff
@@ -0,0 +1,22 @@
+restore() {
+    if [ -z "${SKPR_BACKUP}" ]; then
+        SKPR_BACKUP=$(skpr backup list "${SKPR_ENV}" --json | jq .[0].Name)
+        if [ $? -gt 0 ]; then
+          echo "Failed to get name of most recent backup"
+          exit 1
+        fi
+        # shellcheck disable=SC2086
+        echo "export SKPR_VERSION=$SKPR_VERSION" >> $BASH_ENV
+    fi
+    echo "Deploying backup: ${SKPR_VERSION}"
+
+    # shellcheck disable=SC2086
+    skpr deploy ${SKPR_ENV} ${SKPR_BACKUP}
```

Comment:
Needs to be `skpr restore create`

Reply:
https://github.com/skpr/skpr-orb/blob/8e5c8bf6cea3b2ecf433de0c4d53dd0d2e45f7f3/src/scripts/restore.sh#L19